packages/react-cosmos-config/src/index.js

```diff
@@ -1,11 +1,12 @@
 import path from 'path';
+import slash from 'slash';
 import resolveFrom from 'resolve-from';
 import importModule from 'react-cosmos-utils/lib/import-module';
 
 const resolveUserPath = (userPath, rootPath) =>
-  path.isAbsolute(userPath) ? userPath : (
+  slash(path.isAbsolute(userPath) ? userPath : (
```

Comment:
You don't need to use `slash` for the entire branch. `path.join` already uses the platform specific separator. In this particular case you could call `slash` just for the `userPath`. [Source](https://nodejs.org/api/path.html#path_path_join_paths)